.travis.yml

```diff
@@ -10,6 +10,9 @@ matrix:
     - python: 2.7
     - python: 3.6
 
+ignore:
+  - streamz/dataframe/tests/test_cudf.py
```

Comment:
Why this line?  I would expect the `pytest.importorskip` line to take care of this.